logstash-input-redis.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
 
   s.add_runtime_dependency 'logstash-codec-json'
-  s.add_runtime_dependency 'redis'
+  s.add_runtime_dependency 'redis', '>= 3.0'
```

Comment:
same comment as logstash-plugins/logstash-output-redis#54 (comment)

should this be ~>3 ?

Reply:
Yes! Just pushed up that change. Good catch :)